src/main/resources/resources/content-security-policy.properties

```diff
@@ -1,2 +1,3 @@
+jmix_pageContentSecurityPolicy=Add Content Security Policy at the page level
```

Comment:
```suggestion
jmix_pageContentSecurityPolicy=Replace Content-Security-Policy at the page level
```

Reply:
I used the old label as-is, but makes sense to reword it a bit, thanks!